.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,24 @@
+---
+name: 🐞Bug report
+about: Help us improve by reporting a bug.
+labels: bug
+---
+
+Thank you for reporting a bug with snapshot4s! We appreciate your time and patience.
+
+# The problem
+
+<-- Describe the problem. -->
+
```

Comment:
I think we can also ask for things like snapshot4s version, sbt version, scala version, platform, test framework and if the they use cross version/project matrix

Reply:
That would definitely be useful.